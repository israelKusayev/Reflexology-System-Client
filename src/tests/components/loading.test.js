import React from 'react';
import Loading from '../../components/loading';
import renderer from 'react-test-renderer';
import Root from '../../root';

test('renders correctly', () => {
  const component = renderer.create(
    <Root initialState={{ loading: true }}>
      <Loading />
    </Root>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
