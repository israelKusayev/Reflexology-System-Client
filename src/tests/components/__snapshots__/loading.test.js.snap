// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="loading"
>
  <div
    className="spinner-border"
    role="status"
    style={
      Object {
        "height": "5rem",
        "width": "5rem",
      }
    }
  >
    <span
      className="sr-only"
    >
      Loading...
    </span>
  </div>
</div>
`;
